adm/ecubmcfw#5 Add second EEPROM device in tf307 dts to check zephyr multiple slave i2c driver

From: Konstantin Kirik (snegovick) <snegovick@uprojects.org>

diff --git a/arch/arm64/boot/dts/baikal/bm-mbm.dtsi b/arch/arm64/boot/dts/baikal/bm-mbm.dtsi
index 08d2c5002b5c..b1674063b9de 100644
--- a/arch/arm64/boot/dts/baikal/bm-mbm.dtsi
+++ b/arch/arm64/boot/dts/baikal/bm-mbm.dtsi
@@ -92,11 +92,17 @@ hwmon@52 {
 	};
 
 	eeprom@53 {
-		compatible = "atmel,24c32";
+		compatible = "atmel,24c02";
 		pagesize = <32>;
 		reg = <0x53>;
 	};
 
+  eeprom@73 {
+		compatible = "atmel,24c02";
+		pagesize = <32>;
+		reg = <0x73>;
+	};
+
 	ps2port@54 {
 		compatible = "tp,serio", "tp,tp_serio";
 		reg = <0x54>;
